### PR TITLE
Develop (add feature/template app)

### PR DESCRIPTION
**Pull request by fedeCucchiararo**

### CODE REVIEW

File: staff/fede-gomez/app-template/login.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            <a href="register.html">Register</a>    
+            <a href="login.html">Login</a>     
+    </nav>
+    <section>
+        <h1>Login</h1>
+        <form>
+            <p>User</p>
+            <input name="firstname" type="text">
+            <p>Password</p>
+            <input name="password" type="text">
```
Comment (maryammdot) [inline staff/fede-gomez/app-template/login.html:28]: You could put type = "password". If you do this what the user writes is not visible

File: staff/fede-gomez/app-template/profile.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            <a href="./home.html">Home</a>
+            <a href="news.html">News</a>    
+            <a href="profile.html">Profile</a>    
+            <a href="about.html">About</a>    
+            <a href="contact.html">Contact</a>    
+            <a href="register.html">Register</a>    
+            <a href="login.html">Login</a>       
+    </nav>
+    <section>
+        <h2>Profile</h2>
```
Comment (maryammdot) [inline staff/fede-gomez/app-template/profile.html:23]: Nothing else inside this section? Only profile header?

File: staff/fede-gomez/app-template/login.html (hunk 1 above)
Comment (fedeCucchiararo) [reply staff/fede-gomez/app-template/login.html:28]: Indeed. Thanks for the tip :)

File: staff/fede-gomez/app-template/profile.html (hunk 2 above)
Comment (fedeCucchiararo) [reply staff/fede-gomez/app-template/profile.html:23]: You are right. I'll add a "lorem ipsum" :)

**Pull request merged**